index.js

```diff
@@ -6,11 +6,12 @@ module.exports = DotenvPlugin;
 
 function DotenvPlugin(options) {
   dotenv.config(options || {});
+  this.example = dotenv.parse(fs.readFileSync('./.env.default'));
```

Comment:
Update `'./.env.default'` to `options.sample` so it can read the file you pass in.